backend/src/main/java/com/pickpick/slackevent/application/message/MessageFileShareService.java

```diff
@@ -0,0 +1,101 @@
+
+@Transactional
+@Service
+public class MessageFileShareService implements SlackEventService {
+
+    private static final String EVENT = "event";
+    private static final String USER = "user";
+    private static final String TIMESTAMP = "ts";
+    private static final String TEXT = "text";
+    private static final String CLIENT_MSG_ID = "client_msg_id";
+    private static final String CHANNEL = "channel";
+
+    private final MessageRepository messages;
+    private final MemberRepository members;
+    private final ChannelRepository channels;
+    private final MethodsClient slackClient;
+
+    @Value("${slack.bot-token}")
+    private String slackBotToken;
+
+    public MessageFileShareService(final MessageRepository messages, final MemberRepository members,
+                                   final ChannelRepository channels, final MethodsClient slackClient) {
+        this.messages = messages;
+        this.members = members;
+        this.channels = channels;
+        this.slackClient = slackClient;
+    }
+
+    @Override
+    public void execute(final Map<String, Object> requestBody) {
+        SlackMessageDto slackMessageDto = convert(requestBody);
+
+        String memberSlackId = slackMessageDto.getMemberSlackId();
+        Member member = members.findBySlackId(memberSlackId)
+                .orElseThrow(() -> new MemberNotFoundException(memberSlackId));
+
+        String channelSlackId = slackMessageDto.getChannelSlackId();
+
+        Channel channel = channels.findBySlackId(channelSlackId)
+                .orElseGet(() -> createChannel(channelSlackId));
+
+        messages.save(slackMessageDto.toEntity(member, channel));
+    }
+
+    private Channel createChannel(final String channelSlackId) {
+        try {
+            Conversation conversation = slackClient.conversationsInfo(
+                    request -> request.token(slackBotToken).channel(channelSlackId)
+            ).getChannel();
+
+            Channel channel = toChannel(conversation);
+            channels.save(channel);
+
+            return channel;
+        } catch (IOException | SlackApiException e) {
+            throw new SlackClientException(e);
+        }
+    }
+
+    private Channel toChannel(final Conversation channel) {
+        return new Channel(channel.getId(), channel.getName());
+    }
+
+    private SlackMessageDto convert(final Map<String, Object> requestBody) {
+        Map<String, Object> event = (Map<String, Object>) requestBody.get(EVENT);
+
+        return new SlackMessageDto(
+                (String) event.get(USER),
+                (String) event.get(CLIENT_MSG_ID),
+                (String) event.get(TIMESTAMP),
+                (String) event.get(TIMESTAMP),
+                (String) event.getOrDefault(TEXT, ""),
+                (String) event.get(CHANNEL)
+        );
+    }
+
+    @Override
+    public boolean isSameSlackEvent(final SlackEvent slackEvent) {
+        return SlackEvent.MESSAGE_CREATED == slackEvent;
```

Comment:
```suggestion
        return SlackEvent.MESSAGE_FILE_SHARE == slackEvent;
```
file share enum이 따로 존재하는 것 같아요!!